.teamcity/src/main/kotlin/model/FunctionalTestBucketProvider.kt

```diff
@@ -10,7 +10,25 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.BuildSteps
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
 import java.io.File
 
-val CROSS_VERSION_BUCKETS = listOf(
+/**
+ * QuickCrossVersionTest only tests the last major version.
+ */
+val QUICK_CROSS_VERSION_BUCKETS = listOf(
+    listOf("0.0", "3.0"), // 0.0 <= version < 3.0
+    listOf("3.0", "4.0"), // 3.0 <= version < 4.0
+    listOf("4.0", "5.0"), // 4.0 <= version < 5.0
+    listOf("5.0", "6.0"), // 5.0 <=version < 6.0
+    listOf("6.0", "7.0"), // 6.0 <=version < 7.0
+    listOf("7.0", "8.0"), // 7.0 <=version < 8.0
+    listOf("8.0", "9.0"), // 8.0 <=version < 9.0
+    listOf("9.0", "10.0"), // 9.0 <=version < 10.0
+    listOf("10.0", "11.0"), // 10.0 <=version < 11.0
+    listOf("11.0", "12.0"), // 11.0 <=version < 12.0
+    listOf("12.0", "13.0"), // 12.0 <=version < 13.0
+    listOf("13.0", "99.0") // 13.0 <=version < 99.0
```

Comment:
I don't understand what is going on here. Is this a Gradle version, i.e. is this supposed to be Gradle 13? Or is this a different version?

Do we have some documentation how we are supposed to be running cross version tests? I don't understand how it is supposed to work now.

Reply:
`13.0 <=version < 99.0` means "run cross version tests between Gradle 13 and Gradle 99", which is "nothing" now so it's intended to be an "empty" bucket.

The intention here is to reserve several buckets for future, for example, if we say "7<=Gradle<99", then after Gradle 8.0 we have to do a split again: "7<=Gradle<99" -> "7<Gradle<8" and "8<=Gradle<99".

Anyway, I'll add some explanation here.